docs/source/roadmap.rst

```diff
@@ -16,28 +16,24 @@ contributions. Here's our plans for the next two releases.
     the feature. Pull Requests are open to anyone.
 
 
-3.2
----
-
-* **RHEL/CentOS:** Support RHEL 8.x (assuming it has been released!)
-* **RHEL/CentOS:** Drop support for RHEL/CentOS 6.x.
-* **ChatOps:** Support RBAC.
-* **Orquesta:** Support task retry in workflow definition.
-* **Orquesta:** Support rerun of workflow execution from task(s).
-
 3.3
 ---
 
-* **Ubuntu:** Stop producing Ubuntu 14.04 packages.
+* **RHEL/CentOS:** Drop support for RHEL/CentOS 6.x.
 * **Mistral deprecation:** Orquesta replaces Mistral as the workflow engine.
 
+
 Monitor the `master branch <https://github.com/StackStorm/st2/>`_ to see how we're progressing.
 
 Backlog
 -------
 
-Here's some more things on our list that we haven't scheduled yet:
+Here's some more nice things on our list that we haven't scheduled yet:
 
+* **Python:** Drop Python 2 support, rely on Python 3 ony.
+* **ChatOps:** Support RBAC.
+* **Ubuntu:** Support Ubuntu 20.04 LTS.
```

Comment:
@blag Following your idea, just included Ubuntu 20.04 LTS to the list of items that's not assigned to any release yet.

Reply:
Awesome, thanks!